README.md

```diff
@@ -6,11 +6,12 @@ A Go web service that digitally signs device assertion details.
 The application can be run in two modes: signing service or admin service. Both the web services
 operate under unencrypted HTTP connections, so these should not be exposed to a public network
 as-is. The services should be protected by web server front-end services, such as Apache, that
-provide secure HTTPS connections. Also, the admin service does not include authentication nor
-authorisation, so this service will typically be made available on a restricted network with some
-authentication front-end on the web server e.g. SSO. Typically, the services will only be available
-on a restricted network at a factory, though, with additional security measures, the signing service
-could be made available on a public network.
+provide secure HTTPS connections. Actually, admin service csrf protection forces to be accessed
```

Comment:
I don't think that adding the CSRF sentence here works as it breaks the flow of the point that is being made about security. I think that the comment that you've added about CSRF later is enough - let's remove that comment from here.